ios-task/README.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+This task is intended for people who interview for our iOS Software Engineer roles.
+
+Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting the following tasks.
+Take your time to polish your project. 
+Testing is very important for us, add tests where is possible and design the app in a way that should be easy to be tested.
+
+## iOS News App
+
+Build an iOS application that shows the latest news from the United States, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see a list with the latest **US top headlines news** (see [newsapi.org documentation](https://newsapi.org/docs/)).
+Each news item in the list should have the description of the article, the author, and the image associated to it. 
+If the news doesn't have an image, present an image placeholder of your choice.
+
+If the user clicks on a news item it will navigate to a page displaying the full news article selected.
+
+In the news detail screen the app should also show the number of **likes** and the number of **comments** the news item has.
+That information can be retrieved from two internal not authenticated endpoints: 
+    * `https://cn-news-info-api.herokuapp.com/likes/<ARTICLEID>` 
+    * `https://cn-news-info-api.herokuapp.com/comments/<ARTICLEID>`
```

Comment:
Very cool! Do these just serve random numbers? With a little effort we could move this to a `*.condenast.io` subdomain and host it on Departures. That way it's a little more transparent how it's operating in the future.

Reply:
it creates numbers based on the url, so that same URL same results.
Something like:
```
const hash = (name) => {
  return crypto.createHash("md5").update(name).digest("hex");
};

router.get("/likes/:articlename", async (req, res, next) => {
  const articlename = req.params.articlename;
  const likesnumber = Math.floor(parseInt(hash(articlename), 16) % 100);

  res.json({
    likes: likesnumber,
  });
});

```

Reply:
http://cn-news-info-api.condenast.io is pending DNS approval, let's be sure to change these URLs once we confirm it looks good

Reply:
I assume is `https://` isn't it?
I changed the URLs in the doc, but let's wait for the DNS approval